src/ytdl_sub/prebuilt_presets/helpers/download_deletion_options.yaml

```diff
@@ -0,0 +1,53 @@
+presets:
+
+  "Only Recent":
+    # Only fetch videos after today minus date_range
+    date_range:
+      after: "today-{only_recent_date_range}"
+
+    # Only keep files uploaded after date_range
+    output_options:
+      keep_files_after: "today-{only_recent_date_range}"
+      keep_max_files: "{only_recent_max_files}"
+
+    # Set the default date_range to 2 months
+    overrides:
+      date_range: "2months"  # keep for legacy-reasons
+      only_recent_date_range: "{date_range}"
+      only_recent_max_files: 0
+
+
+  "Fetch Newest":
```

Comment:
Maybe we can call this Only Recent Archive to compliment the shared override vars